src/profile_service/activity.details.js

```diff
@@ -140,6 +140,18 @@ class ActivityDetails {
     return this.receipt.sharing_outcome;
   }
 
+  getErrorDetails() {
+    // eslint-disable-next-line camelcase
+    const errorDetails = this.parsedResponse.error_details;
+    if (errorDetails) {
+      return {
+        errorCode: errorDetails.error_code,
+        description: errorDetails.description,
+      };
+    }
+    return undefined;
```

Comment:
This is redundant

Reply:
Yup, sure, but that makes `eslint` happy... hence present.
Other way would be to write a ternary....

Reply:
or not making the return in the `if` statement, declaring a variable above and returning it no matter what after the `if`...

Reply:
Made the change anyway...